docs/topics/development/setup_and_configuration.md

```diff
@@ -0,0 +1,316 @@
+# Setup and Configuration
+
+## Local Development Environment
+
+Setting up the local development environment for the **addons-server** involves configuring Docker Compose to run the necessary services. Follow these steps to get started:
+
+1. **Prerequisites**:
+    - Ensure Docker and Docker Compose are installed on your system.
+    - Clone the **addons-server** repository from GitHub:
+
+      ```sh
+      git clone https://github.com/mozilla/addons-server
+      cd addons-server
+      ```
+
+2. **Configuration**:
+    - Run the following command to create the `.env` file with necessary environment variables:
+
+      ```sh
+      make setup
```

Comment:
reading onwards, `make initialize_docker` calls `make up`, which then calls `make setup`, so we don't need this step... ?

Reply:
I thought to explicitly call out this command as it is really important to understand in the onboarding process. I think it's enough to later explain that `initialize_docker` and `up` call it but to still explain it and what it does. wdyt
? How would you change to be more clear?

Reply:
imo, as its the first doc new contributors will try to follow, we should keep the steps as minimal as possible, so if `make setup` doesn't need to be called separately, then don't enumerate it as a step.  The explanation here can be moved into "**Understanding `make up`**"

Reply:
Looking again, I agree. There will be another doc detailing the make commands so I'll just leave it out and we can link to it when it's added. I missed this after splitting the other docs.